tsdb/store.go

```diff
@@ -51,6 +51,12 @@ const (
 // a database.
 const SeriesFileDirectory = "_series"
 
+// res holds the result from opening each shard in a goroutine
+type res struct {
+	s   *Shard
+	err error
+}
+
```

Comment:
`res` probably deserves a more descriptive name now that it is in the package scope instead of just the method scope. Alternatively, `enableShards` could be reintegrated into `loadShards` since it is only called once and only in `loadShards`.